system-int-test/test/cmr/system_int_test/search/collection_keyword_search_test.clj

```diff
@@ -183,8 +185,10 @@
           (and parameter-matches? json-matches?))
 
         ;; search by contact persons
-        "Bob Hope" [coll26]
-        "bob.hope@nasa.gov" [coll26]
+        "Bob Hope" [coll26 coll26-1]
+        "\"Bob Hope\"" []
+        "bob.hope@nasa.gov" [coll26 coll26-1]
```

Comment:
While not a deciding factor for me on approving this change, I dislike the use of actual email address with real enough names that could be possible. I looked it up, there is a Richard (by middle name) Hope that works for nasa, but lucky for us he uses his first name so this email is not active. bob.home@example.gov would be better as example.* is the official fake domain name. Same goes for Mr Freeze below. We do have people of the Fries family in NASA, NSIDC could as well even though a google search for this exact email returns no hits, it is possible. I would recommend victor.fries@example.org. Just a suggestion, and maybe it should be in the back of our mind when we create new examples.